ui/public/index.html

```diff
@@ -25,7 +25,7 @@
   <body id='body'>
     <noscript>
       <div style="max-width: 500px; margin: 1rem;">
-        <div><img alt="logo" src="/assets/images/all-of-us-logo.svg" width="155px"></div>
+        <div><img alt="logo" src="/src/assets/images/all-of-us-logo.svg" width="155px"></div>
```

Comment:
I'd have to either eject from create-react-app or pull in react-app-rewired (which I think I want to do anyways because I think it will make it easier to handle having 8 different environments)